app/src/main/res/values/strings.xml

```diff
@@ -928,4 +928,14 @@
   <!-- App update notification -->
   <string name="app_update_notification_subtext">Please update your Simple app in the Google Play Store as soon as possible.</string>
   <string name="critical_app_update_notification_subtext">Important updates available. Update your Simple app from the Google Playstore.</string>
+  
+  <!-- Critical app update dialog -->
+  <string name="critical_update_required_dialog_title">Critical update required</string>
+  <string name="critical_update_required_dialog_content">Update now for critical security changes and latest features</string>
```

Comment:
The text "changes and latest features" should come on the next line as per Figma design. Can you update this?

Reply:
Afaik, the lines that come in the next line can vary based on the screen size. I don't think it would be a good idea to hardcode it to be on the next line. Wdyt? @Honey14

Reply:
For example in the figma design, they have set a particular text size and have set boundaries and not specifically put the text in the next line. Correct me if I'm wrong.

Reply:
I don't think it's that big of a requirement in this case. The design device frame is a 360 dpi based one, so if it's showing 2 lines there, that should be enough. You can confirm with design once @janhavisinghh

Reply:
For example, if we are opening the dialog on a high dpi device. There is going to be a lot of white space on the right side. Showing a single line in that scenario might be a good idea. WDYT?

Reply:
I agree @msasikanth